k8s/aat/common/em/em-stitching.yaml

```diff
@@ -26,6 +26,8 @@ spec:
       image: hmctspublic.azurecr.io/em/stitching:prod-464c8e27
       ingressHost: em-stitching-aat.service.core-compute-aat.internal
       environment:
+        SPRING_DATASOURCE_URL: jdbc:postgresql://em-stitching-postgres-db-aat.postgres.database.azure.com:5432/stitching
+        SPRING_DATASOURCE_USERNAME: annotation@em-stitching-postgres-db-aat
```

Comment:
same here,
https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.yaml#L9

Reply:
not sure why it would go into the draft store?

Reply:
done